### ISSUE TITLE
Appveyor build is broken

### ISSUE THREAD

**Issue by jdm** [servo/servo#10789]
Caused by #10749:
```
    Running `C:/projects/servo/target\debug\build\style-10c01edc6fc2a797\build-script-build`
Build failed, waiting for other jobs to finish...
error: failed to run custom build command for `style v0.0.1 (file:///C:/projects/servo/components/style)`
Process didn't exit successfully: `C:/projects/servo/target\debug\build\style-10c01edc6fc2a797\build-script-build` (exit code: 1)
--- stderr
Traceback (most recent call last):
  File "C:\projects\servo\components\style\properties\build.py", line 15, in <module>
    import data
ImportError: No module named data
```

cc @SimonSapin

**Comment by SimonSapin** [comment]
`build.py` currently relies on this behavior to resolve `import data`:

https://docs.python.org/2/library/sys.html#sys.path
> As initialized upon program startup, the first item of this list, `path[0]`, is the directory containing the script that was used to invoke the Python interpreter.

I don’t believe this behavior is OS-dependent, so I don’t understand why we get this error on Windows/AppVeyor but not other platforms/Buildbot. Nevertherless, I’ll submit a PR to explicitly add the relevant directory to `sys.path`. Hopefully this will help.

**Comment by jdm** [comment]
Fixed by #10791.
